gradle-suppressible-error-prone/src/test/groovy/com/palantir/gradle/suppressibleerrorprone/SuppressibleErrorPronePluginIntegrationTest.groovy

```diff
@@ -301,9 +301,10 @@ class SuppressibleErrorPronePluginIntegrationTest extends IntegrationSpec {
         runTasksSuccessfully('compileAllErrorProne', '-PerrorProneSuppress')
 
         then:
-        runTasksSuccessfully('compileAllErrorProne')
-
         appJavaTextContains('@SuppressWarnings(\"for-rollout:ArrayToString\")')
+
+
+        runTasksSuccessfully('compileAllErrorProne')
```

Comment:
The error messages are better if you just get an IDE powered diff right away rather than "compilation failed, scroll up through output"